lib/menu.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'order'
+require 'yaml'
+
+class Menu
+
+  MENU = YAML::load(File.open('./lib/resources/menu.yml'))
```

Comment:
Style/ColonMethodCall: Do not use `::` for method calls. (https://github.com/bbatsov/ruby-style-guide#double-colons)